### ISSUE TITLE
Process still relevant in 2024?

### ISSUE THREAD

**Issue by PacketAttack0** [mvines/validator-Identity-transition-demo#2]
Hello, thanks for this write up. Is this still relevant on the newest validator code? 1.18.15?

**Comment by mvines** [comment]
Yes. For a bash script that implements what's described here, check out https://github.com/mvines/sosh/blob/master/bin/xferid

**Comment by PacketAttack0** [comment]
great thanks will check this out.
